packages/uniforms-bootstrap3/src/DateField.tsx

```diff
@@ -62,8 +63,8 @@ function Date({
       placeholder={placeholder}
       readOnly={readOnly}
       ref={inputRef}
-      type={dateType}
-      value={dateFormat(value) ?? ''}
+      type={type}
+      value={dateFormat(value, type) ?? ''}
```

Comment:
If we'd inline `dateFormat`, then we could inline `DateFieldType` too, making the `DateField` much simpler.